gm4_orbis/data/gm4_orbis/functions/structure/chest_dungeon.mcfunction

```diff
@@ -6,3 +6,22 @@ execute store result score chest_height gm4_count run data get entity @s Pos[1]
 execute if score chest_height gm4_count matches ..29 run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/dungeon_20"}
 execute if score chest_height gm4_count matches 30..39 run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/dungeon_30"}
 execute if score chest_height gm4_count matches 40.. run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/dungeon_40"}
+
+# set the loot table seed
+data modify storage gm4_orbis:temp Pos set from entity @s Pos
+execute store result score $x gm4_orbis_config run data get storage gm4_orbis:temp Pos[0] 64
+execute store result score $y gm4_orbis_config run data get storage gm4_orbis:temp Pos[1] 64
+execute store result score $z gm4_orbis_config run data get storage gm4_orbis:temp Pos[2] 64
+data remove storage gm4_orbis:temp Pos
+
+scoreboard players operation $loot_table_seed gm4_orbis_config = #world_seed gm4_orbis_config
```

Comment:
Is there a reason why `world_seed` uses a `#` and `loot_table_seed` uses a `$`?

Reply:
`#` are used for constants, `$` are used for non-constant fake players